src/shared/services/reflect.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
this shouldn't be a service, this is more like a set of utils, services are singleton classes handling some feature internally (at least that was my understanding taken from taho extension repo and I think it will be good to continue it here)

It can be wrapped in the service format but if we are not dispatching actions to change the service internal state then I think it is not worth it and we can handle it elsewhere